cmd/influxd/recovery/auth/auth.go

```diff
@@ -120,20 +133,27 @@ func NewAuthCreateCommand() *cobra.Command {
 	cmd.Flags().StringVar(&authCmd.boltPath, "bolt-path", defaultPath, "Path to the BoltDB file")
 	cmd.Flags().StringVar(&authCmd.username, "username", "", "Name of the user")
 	cmd.Flags().StringVar(&authCmd.org, "org", "", "Name of the org")
+	cmd.Flags().BoolVar(&authCmd.useTokenHashing, "store-token-hashes", false, "Store token hashes")
 
 	return cmd
 }
 
-func (cmd *authCreateCommand) run() error {
+func (cmd *authCreateCommand) run() (rErr error) {
 	ctx := context.Background()
 	store := bolt.NewKVStore(cmd.logger.With(zap.String("system", "bolt-kvstore")), cmd.boltPath)
 	if err := store.Open(ctx); err != nil {
 		return err
 	}
-	defer store.Close()
+	defer func() {
+		rErr = errors.Join(store.Close(), rErr)
```

Comment:
For these defers can `errors2.Capture` be used?
see https://github.com/influxdata/influxdb/blob/1efb8dad431b22aea75f9146a17c88141b3296ee/tsdb/index/tsi1/partition.go#L301